### PR TITLE
minor documentation fix

### LINKED ISSUE #7720: Typo/awkward wording in migration documentation guide
This is a really minor issue, and I'm happy to submit a PR for it.

From https://github.com/edgedb/edgedb/blob/9f0195ea2d1b568abf6e56e462734614e1c17419/docs/guides/migrations/guide.rst:
>  It's not unlikely though that you'll define your schema up perfectly with 
 your first try, or that you'll build an application that never needs 
 its schema revised. When you *do* eventually need to make a change, you will 
 need to migrate your schema from its current state to a new state.

The first part of this section has a couple problems:
- The double negative of "It’s not unlikely" implies that it *is* likely that you'll get it perfect the first time
- "define your schema up perfectly with your first try" is awkward phrasing that sounds like a combination of two different phrases

I suggest rewording this section to: (relevant clause in bold)
>  **It's unlikely though that you'll define your schema perfectly 
 your first try**, or that you'll build an application that never needs 
 its schema revised. When you *do* eventually need to make a change, you will 
 need to migrate your schema from its current state to a new state.

### PR DESCRIPTION
**Pull request by willdhorn**
resolves #7720

### CODE REVIEW

**Review by scotttrinh — APPROVED** [review]

**Review by msullivan — APPROVED** [review]

**Comment by msullivan** [comment]
Oops I approved when I meant to merg
